### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by oyamoh-brian**
Added Constructor overloading to make work easier while initialising from a fragment or activity

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="JDK" project-jdk-type="JavaSDK">
```
Comment (ronnieotieno) [inline .idea/misc.xml:3]: Nice pr but please work on this. The changes that arent connected to the pr

File: .idea/gradle.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="testRunner" value="PLATFORM" />
```
Comment (ronnieotieno) [inline .idea/gradle.xml:7]: Please remove the changes that arent related

File: README.md [hunk 3]
```diff
@@ -38,10 +38,10 @@ Example in code, Kotlin:
    override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 	 //activity
-	 imagePicker = ImagePicker(activity = this)
```
Comment (ronnieotieno) [inline README.md:41]: Nice, please do the same with Java example

**Review by ronnieotieno — CHANGES_REQUESTED** [review]
Nice PR, please work on the files I commented, also you can edit the Java snippet in the Read me file

File: .idea/gradle.xml (hunk 2 above)
Comment (oyamoh-brian) [reply .idea/gradle.xml:7]: Noted. You mean the Non-Kotlin Files?

**Comment by ronnieotieno** [comment]
Not really. Just the changes that aren't directly associated with the PR.
Some changes that were introduced by your android studio settings.

**Comment by ronnieotieno** [comment]
Why did you close and delete? The pr was super great

**Pull request closed without merging**